src/main/java/faang/school/accountservice/dto/AccountDto.java

```diff
@@ -0,0 +1,31 @@
+package faang.school.accountservice.dto;
+
+import faang.school.accountservice.enums.AccountStatus;
+import faang.school.accountservice.enums.AccountType;
+import faang.school.accountservice.enums.Currency;
+import faang.school.accountservice.enums.OwnerType;
+import lombok.Builder;
+import lombok.Data;
+import lombok.NonNull;
+
+import java.time.LocalDateTime;
+
+@Data
+@Builder
+public class AccountDto {
+    private long id;
+    private String number;
```

Comment:
It's not a good idea to represent this one in a string format, since account number is kind of a sensitive data. Such things should not be put in the string pool. It's a vulnerable place for this kind of data.

Reply:
Implemented using BigInteger